### PR TITLE
Login activity

### PR DESCRIPTION
**Pull request by rahulramakrishnan3**

### CODE REVIEW

File: src/app/core/services/auth/auth.service.ts [hunk 1]
```diff
@@ -12,13 +12,15 @@ import { UserService } from '../user/user.service';
...
 import { DbService } from '../db/db.service';
+import { UtilService } from '../util/util.service';
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthService {
   baseUrl: any;
   user: any;
+  deviceInfo: string;
```
Comment (Cafnanc) [inline src/app/core/services/auth/auth.service.ts:23]: is device info string? or object

File: src/app/core/services/auth/auth.service.ts [hunk 2]
```diff
@@ -28,15 +30,17 @@ export class AuthService {
     private userService: UserService,
     private profileService: ProfileService,
     private translate: TranslateService,
-    private db: DbService
+    private db: DbService,
```
Comment (Cafnanc) [inline src/app/core/services/auth/auth.service.ts:33]: why dbservice?

File: src/app/core/services/auth/auth.service.ts [hunk 3]
```diff
@@ -84,19 +89,24 @@ export class AuthService {
     return this.user;
   }
 
-  async logoutAccount(skipApiCall?: boolean) {
-    const config = {
+  async logoutAccount(skipApiCall?: boolean, sessionId?: any) {
```
Comment (Cafnanc) [inline src/app/core/services/auth/auth.service.ts:92]: change sessionId to userSessionId

File: src/app/core/services/http/http.service.ts [hunk 4]
```diff
@@ -38,14 +40,22 @@ export class HttpService {
   }
 
   async setHeaders() {
+    const browser = Bowser.getParser(window.navigator.userAgent);
     let token = await this.getToken();
     const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
     const acceptLanguage = await this.localStorage.getLocalData(localKeys.SELECTED_LANGUAGE);
+    const metaData = {
```
Comment (Cafnanc) [inline src/app/core/services/http/http.service.ts:47]: is this needed here?

File: src/app/core/services/http/http.service.ts [hunk 5]
```diff
@@ -38,14 +40,22 @@ export class HttpService {
...
+      osName: browser.getOSName(),
+      platformType: browser.getPlatformType(),
+      type: '',
+    }
     const headers = {
       'X-auth-token': token ? token : "",
       'Content-Type': 'application/json',
       'timeZone': timezone,
-      'accept-language':acceptLanguage
+      'accept-language':acceptLanguage,
```
Comment (Cafnanc) [inline src/app/core/services/http/http.service.ts:58]: revert unwanted change

File: src/app/core/services/http/http.service.ts [hunk 6]
```diff
@@ -23,6 +24,7 @@ export class HttpService {
   baseUrl;
   isFeedbackTriggered = false;
   isAlertOpen: any = false;
+  metaData: { browserName: string; browserVersion: string; osName: string; platformType: string; type: string; };
```
Comment (Cafnanc) [inline src/app/core/services/http/http.service.ts:27]: is this needed now?

File: src/app/core/services/http/http.service.ts (hunk 4 above)
Comment (rahulramakrishnan3) [reply src/app/core/services/http/http.service.ts:47]: i didnt make any change here its already there.

**Pull request merged**